config/config.go

```diff
@@ -84,6 +85,15 @@ type Cookie struct {
 	Value string `mapstructure:"value"`
 }
 
+type Analytics struct {
+	Enabled bool     `mapstructure:"enabled"`
```

Comment:
I think we should delete this as well, and make the root config a `Analytics   *Analytics .  mapstructure:"analytics"` instead.

If a vendor doesn't want analytics, they can omit it from their config.

As written, I think the behavior would be surprising. A vendor who uses the config:

```
analytics:
  file:
    filename: whatever.log
```

almost certainly _intends_ for logs to appear in that file.

But this `Enabled` flag sets it to log nowhere, which would be confusing. ...clearer to just know that "if it's defined, it's active."